FirstSolution/Part11Task1/Task.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace Part11Task1
+{
+    // Написать метод Max() или Min() находящий в массиве максимальное или
+    // минимальное число и возвращающий его. Использовать перегрузку для
+    // работы с типами byte, int, double, float.
+
+    public class Task
+    {
+        private static void Main(string[] args)
+        {
+            Console.WriteLine(Max(new byte[] {1, 4, 3, 2}));
+            Console.WriteLine(Max(new int[] {10, 14, 23, 12}));
+            Console.WriteLine(Max(new double[] {1.6, 4.6, 2.3, 1.2}));
+            Console.WriteLine(Max(new float[] {1.1f, 7.4f, 5.3f, 2.2f}));
+        }
+
+        private static byte Max(byte[] array)
+        {
+            byte max = array[0];
+
+            foreach (var i in array)
+            {
+                if (max < i)
+                {
+                    max = i;
+                }
+            }
```

Comment:
Try to delete brackets here, for foreach and if

Reply:
I know that you could write it this part of code in that way.